core/domain/feedback_services.py

```diff
@@ -291,12 +308,39 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_email_task(user_id)
 
 
-def send_feedback_message_email(exploration_id, thread_id, message_id):
+def update_feedback_email_retries(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+    time = (datetime.datetime.utcnow() - model.created_on).seconds
```

Comment:
utcnow() is the correct thing to use, but I'm not sure if we've ever done a subtraction with created_on before. Good to double check, in any case.